model_analyzer/config/input/config_command_profile.py

```diff
@@ -927,7 +927,7 @@ def _add_run_search_configs(self):
         self._add_config(
             ConfigField(
                 "min_percentage_of_search_space",
-                flags=["--min_percentage_of_search_space"],
+                flags=["--min-percentage-of-search-space"],
```

Comment:
This was uncovered during the L0 test creation. It slipped through unit testing because I copied/pasted into `test_cli.py` and it wasn't caught in the PR